src/System.Configuration.ConfigurationManager/ref/Configurations.props

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp;
-      uap;
+      <!-- uap;  Disabling uap build due to issue: https://github.com/dotnet/corefx/issues/15515 -->
```

Comment:
and the other one

---
In reply to: [98301581](https://github.com/dotnet/corefx/pull/15561#discussion_r98301581) [](ancestors = 98301581)